components/sup/src/event.rs

```diff
@@ -321,6 +331,8 @@ mod tests {
     use futures::{future::Future,
                   stream::Stream,
                   sync::mpsc as futures_mpsc};
+    use habitat_core::service::HealthCheckInterval;
+
```

Comment:
Extra newline.

Reply:
I'm not sure if it's a Rust formatter issue - but when I do not include a new line after that line it tells me it should go under #[cfg(windows)] - which feels incorrect

<img width="1006" alt="Screen Shot 2019-12-18 at 11 24 02 AM" src="https://user-images.githubusercontent.com/813007/71116486-1bde2180-2189-11ea-99e3-07ff03c23223.png">

Reply:
Agreed it looks odd, but it is actually ok. `#[cfg(windows)]` only applies to the immediately followed expression (in this case the line `use habitat_core::os::process::windows_child::ExitStatus;`).

Reply:
Ah! Ty!